src/helpers/analytics.ts

```diff
@@ -18,7 +18,7 @@ import * as pageFocus from './analyticsEvents/pageFocus';
 import * as print from './analyticsEvents/print';
 import * as search from './analyticsEvents/search';
 import * as signupCTA from './analyticsEvents/signupCTA';
-import * as openCloseStudyGuides from './analyticsEvents/studyGuides/openClose';
+import * as openCloseStudyGuides from './analyticsEvents/studyguides/openClose';
```

Comment:
I'm okay with not using came case but at the same time I don't know why we shouldn't do it?